server/src/services/paygov/index.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Reflects the "public" paygov interface to be used by the rest of the application. Provides semantic `startCollection` and `completeCollection` methods to encapsulate internal logic and throw errors in any failure situation. Internal methods are exported from a separate file so they can still be tested.